src/app/pages/network/ipmi/ipmi.component.ts

```diff
@@ -16,9 +16,16 @@ import { T } from '../../../translate-marker';
   selector : 'app-ipmi',
   template : `
   <mat-card class="ipmi-card">
+  <mat-spinner
+    diameter='25'
+    class="form-select-spinner"
+    id="ipmi_controller-spinner"
+    *ngIf="!currentControllerLabel">
+  </mat-spinner>
   <mat-select *ngIf="is_ha" #storageController name="controller" placeholder="Controller" (selectionChange)="loadData()" [(ngModel)]="remoteController">
-    <mat-option [value]="false">Current controller</mat-option>
-    <mat-option [value]="true">Failover controller</mat-option>
+
+    <mat-option [value]="false">Current Controller: TrueNAS Controller {{currentControllerLabel}}</mat-option>
```

Comment:
This is pretty redundant.  "Current" and "Failover" are kind of ambiguous in meaning.  Let's just implement the correct terminology here:
```suggestion
    <mat-option [value]="false">Active: TrueNAS Controller {{currentControllerLabel}}</mat-option>
```